package.json

```
@@ -20,12 +20,16 @@
     "test": "sui-studio test && npm run test:jest",
     "test:jest": "jest"
   },
-  "repository": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SUI-Components/adevinta-spain-components.git"
+  },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "7.10.4",
+    "@s-ui/documentation-library": "1.13.0",
```

```suggestion
    "@s-ui/documentation-library": "1",
```

best, use the mayor version

---

Good catch, I didn't check it, thanks :)